### PR TITLE
Stop converting connection manager properties

### PR DESCRIPTION
**Pull request by KyleAure**
Removed the convert/validateProperty methods in ConnectionManagerServiceImpl and then reconfigured the logic where converted values had been used

### CODE REVIEW

**Comment by KyleAure** [comment]
fixes #2776

**Comment by KyleAure** [comment]
#build

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/ConnectionManagerServiceImpl.java [hunk 1]
```diff
@@ -619,6 +581,15 @@ else if (value instanceof Number)
                     throw resX;
             }
 
+        //Check if immediate is supported.  If not default value?
```
Comment (alexsm82) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/ConnectionManagerServiceImpl.java]: This should probably be `//Check if immediate is supported.  If not throw exception`

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManager.java [hunk 2]
```diff
@@ -2821,8 +2820,9 @@ public String toString2(int options) {
                             aBuffer.append("Connection marked for thread supported cleanup and destroy.  Waiting ");
                             aBuffer.append("for transaction end and connection close - ");
                         } else {
+                            //Changed 0 to -1
```
Comment (alexsm82) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManager.java]: I also assume this comment was intended to be removed

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/FreePool.java [hunk 3]
```diff
@@ -160,15 +160,14 @@ private void queueRequest(ManagedConnectionFactory managedConnectionFactory, lon
         if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled()) {
             Tr.entry(this, tc, "queueRequest", waitTimeout);
         }
-
-        if (waitTimeout < 0) {
+        if (waitTimeout <= 0) {
```
Comment (alexsm82) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/FreePool.java]: This should be `if (waitTimeout == 0)`

File: dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/FreePool.java [hunk 4]
```diff
@@ -160,15 +160,14 @@ private void queueRequest(ManagedConnectionFactory managedConnectionFactory, lon
...
             --pm.waiterCount;
             Tr.error(
                      tc,
                      "POOL_MANAGER_EXCP_CCF2_0001_J2CA0045",
                      new Object[] { "queueRequest", gConfigProps.cfName });
 
-            ConnectionWaitTimeoutException cwte = new ConnectionWaitTimeoutException("Connection not available, Timed out waiting. 0 used for wait timeout");
+            ConnectionWaitTimeoutException cwte = new ConnectionWaitTimeoutException("Connection not available, Timed out waiting. Negative value used for wait timeout");
```
Comment (alexsm82) [inline dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/FreePool.java]: Zero should be used here rather than negative value.

**Comment by alexsm82** [comment]
#build

**Pull request merged**